Core/src/main/java/com/paypal/android/core/GraphQlRequestFactory.kt

```diff
@@ -0,0 +1,23 @@
+package com.paypal.android.core
+
+import org.json.JSONObject
+import java.net.URL
+
+class GraphQlRequestFactory {
```

Comment:
Another nitpick but I think the styling is usually GraphQL with capital "L". Can we update the names of these classes/parameters for consistency?